gnomad_qc/analyses/compute_pext.py

```diff
@@ -118,35 +146,40 @@ def get_pipeline_resources(
     annotation_level = PipelineStepResourceCollection(
         "--annotation-level",
         output_resources={
-            "annotation_pext_ht": get_pext(
-                "annotation_level",
-                gtex_version,
-                vep_version,
-                test=test,
+            "annotation_level_pext_ht": get_pext(
+                "annotation_level", gtex_version, test
             ),
         },
     )
     base_level = PipelineStepResourceCollection(
         "--base-level",
         output_resources={
-            "base_pext_ht": get_pext(
-                "base_level",
-                gtex_version,
-                vep_version,
-                test=test,
-            ),
+            "base_pext_ht": get_pext("base_level", gtex_version, test),
+        },
+    )
+    gnomad_exomes = PipelineStepResourceCollection(
+        "--gnomad-exomes-ht",
+        input_resources={
+            "gnomad exomes HT": {"gnomad_exomes_ht": gnomad_exomes},
+        },
+        output_resources={
+            "gnomad_exomes_pext_ht": get_pext("gnomad_exomes", gtex_version, test),
```

Comment:
`get_pext` already returns `gnomad` in the name, so this will become `f"{_analysis_root('pext', version=version, test=test)}/gnomad.pext.gtex_v10.gnomad_exomes.ht"` -- maybe `name` here could just be `exomes`? and same comment below for genomes